pytest/tests/loadtest/locust/common/linkdrop.py

```diff
@@ -0,0 +1,165 @@
+        drop_id = str(random.randint(1_000_000_000, 10**38 - 1))
+        #Using send_tx_retry gave a lot of errors with drop_id already exists although we are using a random number
+        user.send_tx(InitDrop(self.account, user.account, drop_id),
+                     locust_name="Create Drop Config")
+        return drop_id
+
+
+class InitDropContract(FunctionCall):
+
+    def __init__(self, contract: Account):
+        super().__init__(contract, contract.key.account_id, "new")
+        self.contract = contract
+
+    def args(self) -> dict:
+        return {
+            "owner_id": self.contract.key.account_id,
+            "root_account": "near",
+            "contract_metadata": {
+                "version": "v2",
+                "link": "google.com"
+            }
+        }
+
+    def sender_account(self) -> Account:
+        return self.contract
+
+
+class InitDrop(FunctionCall):
+
+    def __init__(self, contract: Account, account: Account, drop_id: str):
+        super().__init__(account,
+                         contract.key.account_id,
+                         "create_drop",
+                         balance=int(1E23))
+        self.contract = contract
+        self.account = account
+        self.drop_id = drop_id
+
+    def args(self) -> dict:
+        return {
+            "public_keys": [],
+            "deposit_per_use": "100000000000000000000000",
+            "drop_id": self.drop_id
+        }
+
+    def sender_account(self) -> Account:
+        return self.account
+
+
+class AddKey(FunctionCall):
+
+    def __init__(self, linkdrop: Account, sender: Account, public_keys,
+                 drop_id: str):
+        super().__init__(sender,
+                         linkdrop.key.account_id,
+                         "add_keys",
+                         balance=int(15E22))
+        self.linkdrop = linkdrop
+        self.sender = sender
+        self.public_keys = public_keys
+        self.drop_id = drop_id
+
+    def args(self) -> dict:
+        return {"public_keys": self.public_keys, "drop_id": self.drop_id}
+
+    def sender_account(self) -> Account:
+        return self.sender
+
+
+class ClaimDrop(Transaction):
+
+    def __init__(
+        self,
+        sender: Account,
+        new_account_id: str,
+        la_public_key,  #this is the limited access key added to the linkdrop contract already
+        la_secret_key,  #same as above
+        node,
+        balance: int = 0,
+    ):
+        super().__init__()
+        self.sender = sender
+        self.receiver_id = sender.key.account_id
+        self.method = "create_account_and_claim"
+        self.new_account_id = new_account_id
+        # defensive cast to avoid serialization bugs when float balance is
+        # provided despite type hint
+        self.balance = int(balance)
+        self.node = node
+
+        #Create a new key pair, which would be a full access key to your account
+        private_key, public_key = ed25519.create_keypair()
```

Comment:
We have a helper class for this in `pytest/lib/key.py:Key`.
You should be able to do something like:

```python
key = Key.implicit_account()
# Use `key.sk` and `key.pk` as before.
```